flink-table/flink-table-api-java/src/main/java/org/apache/flink/table/api/TableEnvironment.java

```diff
@@ -1252,4 +1254,52 @@ void createFunction(
      * as one job.
      */
     StatementSet createStatementSet();
+
+    // FLIP-190 methods. They're considered experimental and might change in future versions
```

Comment:
nit: at some point FLIP-190 will be history, we don't need this comment here and at other locations. if you wanna structure your code then call this section generic like `Plan compilation and restore`